README.md

```diff
@@ -49,7 +55,29 @@ firestore.updateDocument("FirstCollection/FirstDocument", data)
 
 **Note:** Although you can call `updateDocument` without using `createDocument` to create the document, any documents in your path will not be created and thus you can only access the document by using the path explicitly.
 
-See other library methods [in the wiki](https://github.com/grahamearley/FirestoreGoogleAppsScript/wiki/Firestore-Method-Documentation).
+You can retrieve your data by calling the `getDocument` function:
+
+```javascript
+const dataWithMetadata = firestore.getDocument("FirstCollection/FirstDocument")
+```
+
+You can also retrieve all documents within a collection by using the `getDocuments` function:
+
+```javascript
+const allDocuments = firestore.getDocuments("FirstCollection")
+```
+
+If more specific queries need to be performed, you can use the `query` function followed by an `execute` invocation to get that data:
+
+```javascript
+const allDocumentsWithTest = firestore.query("FirstCollection").where("name", "==", "Test!").execute()
+```
+
+See other library methods and details [in the wiki](https://github.com/grahamearley/FirestoreGoogleAppsScript/wiki/Firestore-Method-Documentation).
+
+### Breaking Changes
+* v16: `createDocumentWithId(documentId, path, fields)` will be **removed** in the next update.
```

Comment:
By this do you mean that method will be removed in v17?

Reply:
Why not? Unless you would like to just remove it immediately. I'm good either way.

Reply:
Yeah, I'm leaning toward just removing it in this PR. I don't think this library is used at a large enough scale that this breaking change would wreak any havoc, so I say let's just do it. 

Good idea to call it out in the readme!